clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -89,6 +107,12 @@ object ClioCommand extends ClioParsers {
   val getServerHealthName = "get-server-health"
   val getServerVersionName = "get-server-version"
 
+  val getSchemaPrefix = "get-schema-"
+  val addPrefix = "add-"
+  val queryPrefix = "query-"
+  val movePrefix = "move-"
+  val deletePrefix = "delete-"
+
   // Names for WGS uBAM commands.
   val getWgsUbamSchemaName = "get-schema-wgs-ubam"
```

Comment:
This could / should be `s"${getSchemaPrefix}-${WgsUbamIndex.commandName}"`. Similarly for the others.